ui/intermittent-failures/GraphAlternateView.jsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable array-callback-return */
```

Comment:
In general we try to disable the rule unless its poorly written. I'm curious where you hit this rule.